core/src/test/scala/com/microsoft/azure/synapse/ml/nbtest/SynapseUtilities.scala

```diff
@@ -131,8 +137,8 @@ object SynapseUtilities extends HasHttpClient {
   }
 
   def poll(id: Int, livyUrl: String): LivyBatch = {
-    val getStatusRequest = new HttpGet(s"$livyUrl/$id")
-    getStatusRequest.setHeader("Authorization", s"Bearer $Token")
+    val getStatusRequest = new HttpGet(s"${livyUrl}/${id}")
```

Comment:
nit: you dont always need these parens and intellij might actually complain if they are unecessary, though it depends on the author which they prefer. I dont really care that much either way but food for thought

Reply:
I liked the readability with parens -- but its more a style choice --- whatever works for the codebase -- I'd just to be normalized one way or another. Im assuming your preference is not to have 'em ?